CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This file documents all notable changes to the GEOS-Chem repository starting in
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased] - TBD
+### Added
+- Added the `KPP_INT_AUTOREDUCE` C-preprocessor switch integrator-specific handling
```

Comment:
I know it would be long, but could this be KPP_INTEGRATOR_AUTOREDUCE? I read INT as integer.

Reply:
Now fixed in commit [8c306d5](https://github.com/geoschem/geos-chem/pull/2689/commits/8c306d5957d41d703677a58090bf44c53b2e10ee)